spec/rails_helper.rb

```diff
@@ -54,3 +54,5 @@
   # https://relishapp.com/rspec/rspec-rails/docs
   config.infer_spec_type_from_file_location!
 end
+
+require "pry"
```

Comment:
The location of this line doesn't matter code-wise, but it matters style-wise. Move it up where the other require statements are in the file.